src/modules/pageBody.js

```diff
@@ -0,0 +1,175 @@
+        <i class="fa-solid fa-xmark"></i>
+      </button>
+      <div class="card-details">
+        <img src="${movieDetails.image.original}" alt="Card original image" />
+        <h3>${movieDetails.name}</h3>
+        <ul class="more-info">
+          <li>
+            <span><strong> Language </strong>: ${movieDetails.language}</span>
+            <span><strong> Genre </strong>: ${movieDetails.genres}</span>
+          </li>
+          <li>
+            <span><strong> Runtime </strong>: ${movieDetails.runtime} minutes</span>
+            <span><strong> premiered </strong>: ${movieDetails.premiered}</span>
+          </li>
+        </ul>
+        <p class="summary">${movieDetails.summary}</p>
+        <div class="card-comment">
+          <h3 class="comm-header"></h3>
+          <div class="comment-area">
+            <div class="comment-list">
+            </div>
+            <form autocomplete="off" id="comment-form">
+              <input type="text" class="username" name="name" id="name" placeholder="Your name"/>
+              <textarea name="message" class="usermassage" id="" cols="30" rows="5" placeholder="Your Insight" ></textarea>
+              <button class="new-comment" id="${btnId}" type="button">Comment</button>
+            </form>
+          </div>
+        </div>
+      </div>
+    </div>
+      `;
+
+      commentModal.innerHTML = modal;
+      commentModal.style.display = 'block';
+      closePopupModal();
+
+      const cmntBtn = document.querySelector('.new-comment');
+      cmntBtn.addEventListener('click', () => {
+        postComment(btnId);
+      });
+    });
+  });
+};
+
+// Render all Movies card
+const renderMovies = async () => {
+  const arr = await fetchFromApi();
+  arr.slice(0, 10).forEach((movie) => {
+    fetchMovieLikes().then((res) => {
+      currentValue = res;
+      let assignLike = 0;
+
+      const like = currentValue.filter(
+        (element) => element.item_id === movie.name,
+      );
+      if (like.length === 0) {
+        assignLike = 0;
+      } else {
+        assignLike = like[0].likes;
+      }
+      moviesCards.innerHTML += `
+        <div class="main-card" id="${arr.indexOf(movie)}">
+          <img src="${movie.image.medium}" alt="${movie.name}" />
+          <div class="list-movies">
+            <h2 class="title">${`${movie.name.substring(0, 15)}`}</h2>
+            <div>
+              <button type="button" class="like"><i class="fa-solid fa-thumbs-up"></i><span id="count-like">${assignLike}</span></button>
+            </div>
+          </div>
+          <div class="card-text">        
+            <p>${`${movie.summary.substring(0, 80)} ...`}</p>
+          </div>
+            <button type="button" class="comment-btn" id="${arr.indexOf(
+    movie,
+  )}">Read more & comments</button>
+        </div>`;
+      showCommentModal();
+    });
+
+    count += 1;
+  });
+
+  counter.innerText = `(${count * 2})`;
+  counter.style.color = 'red';
```

Comment:
- [ ] As per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=Even%20if%20the%20API%20gives%20you%20these%20numbers%2C%20you%20will%20create%20a%20specific%20function%20to%20calculate%20these%20numbers%20in%20each%20page.%20These%20count%20functions%20need%20to%20be%20covered%20with%20unit%20tests%20using%20Jest.) requirement, please make sure that there's a function that on its own calculate the number of items on the page and then display it. This will open up the possibility of testing it as a pure function without mocking it. This also abies by the SOLID principle.